config/pipeline.yaml

```diff
@@ -1865,6 +1865,18 @@ jobs:
       <<: *ltp-params
       tst_cmdfiles: "cap_bounds"
 
+  ltp-containers:
+    <<: *ltp-job
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "containers"
```

Comment:
All the LTPs added by this patch will be submitted to KCIDB as "ltp" suite:
https://github.com/kernelci/kernelci-pipeline/blob/c4bb6cdc41e80b070c02db5bfe0ffa67c1d8dd7b/config/pipeline.yaml#L86

@broonie Do you plan to use KCIDB as a data source for processing the results? Should the LTP Jobs' parameters be extended with `kcidb_test_suite` like the kselftests?

Reply:
Yeah, I was wondering about the decision to not use subsuites for LTP, I was just following what was there for the existing LTP entries. One reason for keeping things that way is that people make their own custom LTP test lists and run those so tests end up showing up in multiple groupings, or different groupings per CI system.

Reply:
Got it - let's keep LTPs as they were for now and raise the topic of test suite naming in the KCIDB submissions at the next weekly call with @spbnick (I'll add it to the agenda)